include/ofi_proto.h

```diff
@@ -50,7 +50,8 @@ extern "C" {
 
 /* ofi_ctrl_hdr::type */
 enum ofi_ctrl_type {
-	ofi_ctrl_connreq,
+	ofi_ctrl_none = -1, /* -1 for compat with older remote versions*/
```

Comment:
This is a protocol value.  None for this doesn't make sense.  Please avoid adding non-wire values here.